packages/plugins/embed/src/utils/providers.ts

```diff
@@ -31,7 +31,7 @@ export function getProvider(url: string): EmbedProviderTypes | null {
     return 'vimeo';
   } else if (url.includes('dailymotion.com') || url.includes('dai.ly')) {
     return 'dailymotion';
-  } else if (url.includes('twitter')) {
+  } else if (url.includes('twitter') || url.includes('https://x.com')) {
```

Comment:
url.includes('x.com') is not enough, that will also treat `matrix.com` also as twitter, so i added the https prefix.